runner/wait.bash

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
We should probably be consistent with the shebang across all the scripts, either use the env bash or the hard coded bash path:

```suggestion
#!/bin/bash
```

Reply:
Thanks! Addressed in f65ec10